dbms/src/AggregateFunctions/AggregateFunctionMinMaxAny.h

```diff
@@ -243,6 +257,17 @@ struct SingleValueDataString : public CommonImpl
             static_cast<ColumnString &>(to).insertDefault();
     }
 
+    void batchInsertSameResultInto(IColumn & to, size_t num) const
+    {
+        if (has())
+        {
+            for (size_t i = 0; i < num; ++i)
```

Comment:
I think you can add some specific function in `ColumnString` instead of use for loop?

Reply:
> I think you can add some specific function in `ColumnString` instead of use for loop?

done